upstream-community-operators/postgresql/postgresql.package.yaml

```diff
@@ -2,12 +2,4 @@ packageName: 'postgresql'
 defaultChannel: stable
 channels:
   - name: stable
-    currentCSV: 'postgresoperator.v4.5.1'
-  - name: original_40
-    currentCSV: 'postgresoperator.v4.0.1'
-  - name: original_41
-    currentCSV: 'postgresoperator.v4.1.0'
```

Comment:
If you want to remove this channel then you have to put relaces to all versions that they are replacing previous. Version `4.2.0` is not replacing `4.1.0` and `4.1.0` is not replacing `4.0.1`. Please fix replaces or leave `original_40` and `original_41` channels in package file

Reply:
@mvalarh Thanks. Let's keep it simple. I'll revert that back to the original change, and I'll push up the modification momentarily.